src/utilities/Button.tsx

```diff
@@ -2,14 +2,16 @@ import { ButtonHTMLAttributes } from "react";
 
 type Props = {
```

Comment:
For Components like button you extends you Component Props with ButtonHTMLAttributes from react library.